Solutions/VCenter/Data Connectors/Connector_Syslog_vcenter.json

```diff
@@ -0,0 +1,138 @@
+                        { 
+                        "parameters": { 
+                        "linkType": "InstallAgentOnLinuxNonAzure" 
+                        }, 
+                        "type": "InstallAgent" 
+                        } 
+                    ] 
+                } 
+            ] 
+                }, 
+                    "type": "InstructionStepsGroup" 
+                    } 
+                ] 
+                    }, 
+    { 
+            "title": "2. Configure the logs to be collected", 
+            "description":"Follow the configuration steps below to get vCenter server logs into Azure Sentinel. Refer to the [Azure Monitor Documentation](https://docs.microsoft.com/azure/azure-monitor/agents/data-sources-json) for more details on these steps.\n For vCenter Server logs, we have issues while parsing the data by OMS agent data using default settings. \nSo we advice to capture the logs into custom table **vCenter_CL** using below instructions. \n1. Login to the server where you have installed OMS agent.\n2. Download config file [vCenter.conf](https://aka.ms/sentinel-vcenter-conf) \n\t\twget -v https://aka.ms/sentinel-vcenteroms-conf -O vcenter.conf \n3. Copy vcenter.conf to the /etc/opt/microsoft/omsagent/**workspace_id**/conf/omsagent.d/ folder. \n\t\tcp vcenter.conf /etc/opt/microsoft/omsagent/<<workspace_id>>/conf/omsagent.d/\n4. Edit vcenter.conf as follows:\n\n\t a. vcenter.conf uses the port **22033** by default. Ensure this port is not being used by any other source on your server\n\n\t b. If you would like to change the default port for **vcenter.conf** make sure that you dont use default Azure monotoring /log analytic agent ports I.e.(For example CEF uses TCP port **25226** or **25224**) \n\n\t c. replace **workspace_id** with real value of your Workspace ID (lines 14,15,16,19)\n5. Save changes and restart the Azure Log Analytics agent for Linux service with the following command:\n\t\tsudo /opt/microsoft/omsagent/bin/service_control restart\n6. Modify /etc/rsyslog.conf file - add below template preferably at the beginning / before directives section \n\t\t$template vcenter,\"%timestamp% %hostname% %msg%\\n\" \n7. Create a custom conf file in /etc/rsyslog.d/ for example 10-vcenter.conf and add following filter conditions.\n\n\t With an added statement you will need to create a filter which will specify the logs coming from the vcenter server to be forwarded to the custom table.\n\n\t reference: [Filter Conditions — rsyslog 8.18.0.master documentation](https://rsyslog.readthedocs.io/en/latest/configuration/filters.html)\n\n\t Here is an example of filtering that can be defined, this is not complete and will require additional testing for each installation.\n\t\t if $rawmsg contains \"vcenter-server\" then @@127.0.0.1:22033;vcenter\n\t\t & stop \n\t\t if $rawmsg contains \"vpxd\" then @@127.0.0.1:22033;vcenter\n\t\t & stop\n\t\t \n8. Restart rsyslog\n\t\t systemctl restart rsyslog",
```

Comment:
Line numbers are not correct. Modify to 13,14,15,18

Reply:
change completed